package.json

```diff
@@ -1,50 +1,53 @@
 {
-  "name": "de.appplant.cordova.plugin.local-notification",
-  "cordova_name": "Cordova LocalNotification Plugin",
-  "version": "0.8.9-OS6",
+  "name": "cordova-plugin-local-notification",
+  "version": "0.9.0-beta.3",
```

Comment:
Why are we using this version?

Reply:
Simply pulled the latest release they had.
Do you suggest otherwise?

Reply:
@cmfsotelo doesn't this repository contain custom changes over the upstream? If so, the new version should be 0.9.0-beta.3-OS

Reply:
The concern is the missing -OS part? 
i did not raise the version because i thought that is done on the branch itself and not on the PR.
Should i place it in the PR?

Reply:
My concern was that we were applying changes over a beta version. And we already discussed that offline, yesterday.